corehq/apps/app_manager/templates/app_manager/managed_app.html

```diff
@@ -102,21 +102,39 @@
     </script>
     {% endif %}
     <script>
-        $(function () {
-            setTimeout(function () {
```

Comment:
This was a hack to bind to an element that gets loaded after the `ready` event. jQuery delegation can handle this for us (see below)